app/reddit_actions.py

```diff
@@ -26,11 +31,29 @@ def submit_transcription(
     reddit_submission = request.user.reddit.submission(url=submission_obj.url)
     transcription = reddit_submission.reply(transcription_obj.text)
     transcription_obj.original_id = transcription.fullname
-    transcription_obj.url = transcription.permalink
+    transcription_obj.url = BASE_URL + transcription.permalink
     transcription_obj.removed_from_reddit = False
     transcription_obj.save()
 
 
+def edit_transcription(
+    request: HttpRequest, transcription_obj: Transcription, submission_obj: Submission
+) -> None:
+    """Post the updated transcription as an edit to the existing comment."""
+    url = transcription_obj.url
+    if not url:
+        # Something went wrong; it was never pushed in the first place.
+        # Try it again.
+        submit_transcription(request, transcription_obj, submission_obj)
```

Comment:
Won't URL still be `None` after we execute this?

Reply:
you're right, it will be None for a few CPU cycles :smile: I added a return right after. Great catch!